### DISCUSSION TITLE
installing radare2 on 32bit windows failed

### DISCUSSION THREAD

**Discussion by AirShoot125** [radareorg/radare2#21360] [Q&A]
I downloaded and tried to run radare2 on 32 bit windows in cmd, what's wrong? Seems like version are correct (if translate - computer says that this is not 32 bit app [IMG_20230219_195328.jpg](https://user-images.githubusercontent.com/114504779/219963018-5b2845a1-3e44-4a91-be46-4862052e4f3c.jpg)

**Comment by trufae** [comment]
did you tried with the 5.8.4 build? i dont have any windows xp or 32bit machine to test but will be good to find out the reason, because the CI is building it properly :? also did you tried building it by yourself if you have msvc it should work. i'll try to get crosscompiled zig cc builds at some point, but this sounds a little less stable than what msvc should generate
